submissions/ogennaOgbogu/HTML Forms/contactForm.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Contact Form</title>
+    </head>
+
+<body>
+    <h1>Contact Me</h1>
+    <form action="#" method="post" enctype="multipart/form-data">
+        <fieldset>
+            <legend>Contact Form</legend>
+            
+            <label for="fname">Full Name</label>
+            <input type="text" name="fname" id="fname"/><br>
+            
+            <br>
+
+            <label for="email">Email</label>
+            <input type="email" name="email" id="email" placeholder="Enter Email here"/>
+
+            <br><br>
+
+            <label for="subject">Subject:</label><br>
+            <textarea name="subject" id="subject" cols="30" rows="10">
+
+            </textarea>
+
+            <label for="message">Message:</label><br>
+            <textarea name="message" id="message" rows= "10" cols = "30"> 
+
+            </textarea><br><br>
+
```

Comment:
remove too many newline spaces

Reply:
Okay, thank you for your suggestions, effecting them now.